opensrp-chw-malaria/src/main/res/layout/activity_malaria_profile.xml

```diff
@@ -0,0 +1,158 @@
+                        style="@style/CustomFontTextViewStyle.ClientList"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center_horizontal"
+                        android:layout_marginTop="5dp"
+                        android:paddingStart="0dp"
+                        android:paddingEnd="0dp"
+                        android:text="@string/patient_name_age"
+                        android:textColor="@android:color/black"
+                        android:textSize="@dimen/register_member_name_size"/>
+
+                <LinearLayout
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center_horizontal"
+                        android:layout_marginTop="10dp"
+                        android:layout_marginBottom="10dp"
+                        android:gravity="center"
+                        android:orientation="horizontal">
+
+                    <org.smartregister.view.customcontrols.CustomFontTextView
+                            android:id="@+id/textview_gender"
+                            style="@style/CustomFontTextViewStyle.ClientList.Light"
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:gravity="center_horizontal"
+                            android:text="@string/gender"
+                            android:textAppearance="@style/TextAppearance.AppCompat.Small"
+                            android:textColor="@android:color/black"
+                            android:textSize="@dimen/register_member_village_size"/>
+
+                    <TextView
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:layout_gravity="center_vertical"
+                            android:layout_marginLeft="@dimen/profile_dot_separator_margin"
+                            android:layout_marginRight="@dimen/profile_dot_separator_margin"
+                            android:text="@string/collon"
+                            android:textColor="@android:color/black"/>
+
+                    <org.smartregister.view.customcontrols.CustomFontTextView
+                            android:id="@+id/textview_address"
+                            style="@style/CustomFontTextViewStyle.ClientList.Light"
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:gravity="center_horizontal"
+                            android:paddingStart="0dp"
+                            android:paddingEnd="0dp"
+                            android:text="@string/address"
```

Comment:
fetch from DB

Reply:
placeholder, i needed to add this things to make sure my views were fine. But it is being replaced with data from DB

Reply:
> placeholder, i needed to add this things to make sure my views were fine. But it is being replaced with data from DB

but we fixed the sample app so that you can use for test

Reply:
this was done before the fix